modules/nw-infw-operator-cr.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * networking/ingress-node-firewall-operator.adoc
+
+:_content-type: PROCEDURE
+[id="nw-infw-operator-cr_{context}"]
+= Ingress Node Firewall Operator
+
+The Ingress Node Firewall Operator deploys daemon on OVN-Kubernetes cluster network provider clusters. To deploy the DaemonSet, you create an `IngressNodeFirewallConfig` custom resource (CR). The Ingress Node Firewall Operator applies the `IngressNodeFirewallConfig` CR to create ingress node firewall daemonset `daemon`, which run on all nodes that match the `nodeSelector`.
+
+You configure the `IngressNodeFirewallConfig` through the `IngressNodeFirewall` CR. The `IngressNodeFirewall` applies the `sourceCIDR` and `rules` you set up to clusters with `nodeSelector` `matchLabels` values set to `true` for `do-node-ingress-firewall`.
```

Comment:
labels can be anything we used do-node-ingress-firewall just as an example but users can configure whatever label they like